spec/services/linked_data/graph_service_spec.rb

```diff
@@ -139,39 +121,56 @@
         end
       end
 
-      context 'when multiple languages passed in as strings in an array' do
-        let(:language) { ["en", "fr"] }
```

Comment:
This was testing normalization of language. That processing moved to LanguageService and is tested there.